src/setup-bot.ts

```diff
@@ -73,6 +79,16 @@ export async function setupBot (): Promise<void> {
     }),
   )
 
+  // Auto load Wechaty Chatopera Plugin with ENV variables
+  if ((process.env['WCP_DEFAULT_CLIENTID'] && process.env['WCP_DEFAULT_SECRET']) || process.env['WCP_PERSONAL_ACC_TOKEN']) {
+    haWechaty.use(
+      WechatyChatopera({
```

Comment:
I believe a good behavior plugin should only accept a config from its args, instead of reading the environment variable?

See my comment at https://github.com/wechaty/wechaty-chatopera/issues/6#issuecomment-933139726

Reply:
Fixed, wechaty-chatopera is configured with params in config now. No ENV values are read in chatopera plugin. So, the checks are done with setup-bot.ts in osschat.